chanreg.md

```diff
@@ -1,228 +1,196 @@
+network. Formal group registration is **not** required to make use of channels
+in the on-topic project and community channel namespace, and we recommend that
+new initiatives do not immediately attempt a formal group registration.
+
+## Channel namespaces
 
-Informal channels that are not already taken can be registered by any user in
-it who has channel operator status, which is usually the first user to join an
-empty, unregistered channel. In this case you can use the `REGISTER` command
-in chanserv to register it to your account. See `/msg ChanServ help REGISTER`
-for details. If a channel is unregistered but nobody has operator status,
-please contact network staff and we shall see if we can find a solution that
-is acceptable for the local community.
-
-## Project registration
-
-Project registration allows your project or organisation to have an official
-representation on our services and relationship with us.
+Our [policies](/policies) outline that channels on Libera.Chat fall into one
+of three categories and two namespaces. Channels that begin with only a single
+`#` character are for on-topic project and community channels (e.g. `#linux`).
+Channels that begin with two or more `#` characters are for informal channels
+(e.g. `##chat`).
 
-### On-topic projects
+### What is on-topic?
 
 Projects considered to be on-topic for Libera.Chat are primarily free and
 open-source software projects, and other peer-directed projects, for instance
 Linux User Groups (LUGs), student societies, hacker- or makerspaces and other
-collaborative efforts. It can also cover projects/companies of general
-interest to our user base.
-
-### Claim to the name and representation
-
-A claim to the name is given if your project is on-topic as per the above and
-named like that, or commonly called / abbreviated that. In case of name
-clashes, e.g. other on-topic projects already present using the same name or
-there being obvious trademark issues, our team will try to find a good
-solution. The same applies if your name contains characters that can't be used
-on our services for technical reasons.
-
-The registration should be either done by or have received the official
-blessing of someone who is able to speak for the project. This could be:
+collaborative efforts. At staff discretion, it can also cover projects or
+companies of general interest to our user base.
+
+If your channel fits the above criteria and you are or have the official
+blessing of someone who is able to speak for the project or group, then you
+would qualify for an on-topic project channel. If your channel fits the above
+criteria but is otherwise unofficial, then you would qualify for an on-topic
+community channel. If your channel does not fit the above criteria, and it is
+not otherwise
+[off-topic for Libera.Chat](/policies/#unwanted-content-and-behaviour), then
+you would qualify for an informal channel.
+
+### Expiration and disputes
```

Comment:
Things on my mind while reading this:

- Staff reserve the right to reassign channels both in `#` and `##` registration or not
- When external projects want to claim channels this happens in the `#` namespace, making `##` a safer place to claim for unofficial stuff
- `##` channels become eligible for reassignment merely due to inactivity
- We don't _give out_ informal channels (which reads to me like something we actively do. All channels are claimed through ChanServ. The first person to register a channel becomes its founder.
- Channel reassignment isn't something done lightly in any of the cases

Reply:
Added a block about reassignment and cleaned up other wording. The text already mentions the inactivity expiration for `##` channels, so I don't think we need additional wording there. I'm also hesitant to outright suggest registering `##` over `#` as a throwaway sentence here because the determination should really be based on whether it's on-topic or not, not whether you think an external entity is going to drop in and swoop your channel away from you (which given this is the "Expiration and disputes" section, would have as an implied context even if not explicitly stated).